benchmarks/project/src/main/java/io/camunda/zeebe/App.java

```diff
@@ -80,6 +81,9 @@ void printTopology(final ZeebeClient client) {
         break;
       } catch (final Exception e) {
         // retry
+        if ((failureCount++ % 1000) == 0) {
```

Comment:
:wrench: Maybe `ThrottledLogger` is the right tool for this job?

Reply:
Ah thanks @oleschoenburg wasn't aware of it